course/lesson-18-for-loops/lesson.tres

```diff
@@ -122,9 +122,49 @@ title = ""
 type = 0
 text = "In the above example, for each item in the list [code][0, 1, 2][/code], Godot sets [code]number[/code] to the item, then executes the code in the [code]for[/code] loop.
 
-In this example, we're printing the value of [code]number[/code] as Godot moves through the loop.
+We'll get to explain arrays more throughly in the next lesson, but pay notice on the fact that [code]number[/code] is just a variable you are assigning values to using the [code]for[/code] loop, and you could name it anything you want. The above code has the exact same behaviour as the next example:"
```

Comment:
pay notice of the fact that -> redundant, maybe incorrect, but for conciseness, just `notice that` is good. Avoid extra words like "more" or "just"

Reply:
Other conciseness examples: exact same -> same

Reply:
> The above code has the exact same behaviour as the next example

Edited:

> This code behaves the same as the previous example